tasks/verify-enterprise-contract/0.1/README.md

```diff
@@ -11,23 +11,21 @@ This task verifies a signature and attestation for an image and then runs a poli
 kubectl apply -f https://raw.githubusercontent.com/conforma/cli/main/tasks/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml
 
 ## Parameters
-### Required
-* **IMAGES**: A JSON formatted list of images.
-### Optional
-* **POLICY_CONFIGURATION**: Name or inline policy in JSON configuration to use. For name `namespace/name` or `name` syntax supported. If
-        namespace is omitted the namespace where the task runs is used. For inline policy provide the [specification](https://conforma.dev/docs/ecc/reference.html#k8s-api-github-com-enterprise-contract-enterprise-contract-controller-api-v1alpha1-enterprisecontractpolicyspec) as JSON.
-* **PUBLIC_KEY**: Public key used to verify signatures. Must be a valid k8s cosign
-        reference, e.g. k8s://my-space/my-secret where my-secret contains
-        the expected cosign.pub attribute.
-* **REKOR_HOST**: Rekor host for transparency log lookups
-* **SSL_CERT_DIR**: Path to a directory containing SSL certs to be used when communicating
-        with external services.
-* **CA_TRUST_CONFIGMAP_NAME**: The name of the ConfigMap to read CA bundle data from.
-* **CA_TRUST_CONFIG_MAP_KEY**: The name of the key in the ConfigMap that contains the CA bundle data.
-* **STRICT**: Fail the task if policy fails. Set to "false" to disable it.
-* **HOMEDIR**: Value for the HOME environment variable.
-* **EFFECTIVE_TIME**: Run policy checks with the provided time.
-* **WORKERS**: Number of parallel workers to use for validation.
+
+* **IMAGE_URL**: The URL of the container image to validate
+* **IMAGE_DIGEST**: The digest of the container image to validate  
+* **POLICY_CONFIG**: The policy configuration to use for validation (default: "github.com/enterprise-contract/ec-release-policy:latest")
+* **POLICY_CONFIG_DATA**: Additional policy configuration data (default: "github.com/enterprise-contract/ec-release-policy:latest")
+* **EFFECTIVE_TIME**: The effective time for policy evaluation (default: "2023-01-01T00:00:00Z")
+* **OUTPUT**: The output format for validation results (default: "json")
+* **SHOW_SUCCESSES**: Whether to show successful validations (default: "false")
+* **VERBOSE**: Enable verbose output (default: "false")
+* **TIMEOUT**: Maximum execution time (default: "5m")
+* **RETRY_DURATION**: Base duration for exponential backoff calculation (default: "1s")
+* **RETRY_FACTOR**: Exponential backoff multiplier (default: "2")
+* **RETRY_JITTER**: Randomness factor for backoff calculation (default: "0.1")
+* **RETRY_MAX_RETRY**: Maximum number of retry attempts (default: "3")
+* **RETRY_MAX_WAIT**: Maximum wait time between retries (default: "3s")
```

Comment:
Did we nuke some of the existing readme content?

Reply:
I see duplicate parameters that I need to fix.

Reply:
This should be fixed now.